test/unit/logic/communities.spec.ts

```diff
@@ -344,22 +346,16 @@ describe('Community Component', () => {
 
   describe('when getting public communities', () => {
     const options = { pagination: { limit: 10, offset: 0 }, search: 'test' }
-    const mockCommunities = [
+    const mockCommunities: Omit<CommunityPublicInformation, 'ownerName'>[] = [
       {
         id: communityId,
         name: 'Test Community',
         description: 'Test Description',
         ownerAddress: '0x1234567890123456789012345678901234567890',
-        privacy: 'public' as const,
+        privacy: CommunityPrivacyEnum.Public,
         active: true,
-        role: CommunityRole.Member,
         membersCount: 10,
-        isHostingLiveEvent: false,
-        voiceChatStatus: {
```

Comment:
Is there a reason this part of the mock was removed?

Reply:
yes, they don't belong to that type. I've added the type and it started complaining about these extra properties which aren't expected:
```typescript
export type CommunityPublicInformation = Omit<CommunityWithUserInformation, 'role' | 'friends' | 'privacy'> & {
  privacy: CommunityPrivacyEnum.Public
}
```